src/crypto/tls/handshake_client.go

```diff
@@ -23,6 +23,31 @@ import (
 	circlSign "circl/sign"
 )
 
+// EXP_EventClientHandshakeTimingInfo is carries intra-stack durations for
```

Comment:
This event will only be emitted during a TLS 1.3 handshake, right? I suggest adding "TLS13" to the name.